emojiHeatmapper/index.html

```diff
@@ -10,6 +10,18 @@
   <link rel="stylesheet" href="http://openlayers.org/en/v3.12.1/css/ol.css" type="text/css">
   <!-- OpenLayers JS -->
   <script src="http://openlayers.org/en/v3.12.1/build/ol.js" type="text/javascript"></script>
+  <script type="text/JavaScript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
+  <script type="text/JavaScript">
+    $(document).ready(function(){
+      var listItems= "";
+      $.getJSON("emoji.json", function(json) {
+      for (var i = 0; i < json.data.length; i++){
+        listItems+= "<option value='" + json.data[i][1] + "'>" + json.data[i][1] + "</option>";
```

Comment:
Good suggestion though, but if I change the value to integer the script.js file should also be changed, because the value of the field is sent as query to Loklak api. I can make the changes in the next iteration if you think so. 👍